python/examples/sample-selection.py

```diff
@@ -0,0 +1,74 @@
+import sys
+
+import chemfiles
+
+from rascaline import SoapPowerSpectrum
+from equistore import Labels, TensorMap
+import numpy as np
+
+with chemfiles.Trajectory(sys.argv[1]) as trajectory:
+    frames = [f for f in trajectory]
+
+HYPER_PARAMETERS = {
+    "cutoff": 5.0,
+    "max_radial": 6,
+    "max_angular": 4,
+    "atomic_gaussian_width": 0.3,
+    "center_atom_weight": 1.0,
+    "radial_basis": {
+        "Gto": {},
+    },
+    "cutoff_function": {
+        "ShiftedCosine": {"width": 0.5},
+    },
+}
+
+calculator = SoapPowerSpectrum(**HYPER_PARAMETERS)
+
+descriptor = calculator.compute(frames)
+
+# The names of the samples that will be chosen during training
+# coincide with the samples of the blocks obtained after calculating the
+# properties. Accordingly, the names of the samples can be obtained with the
+# following commands (usually 'structure' (structure number) and 'center'
+# (atom number in the structure).
+print(f"samples names: {descriptor.block(0).samples.names}")
+
+# selected_samples can be a variable of `Labels` or `TensorMap` type.
+# In the first case, if we want to select first four structures:
+labels = Labels(
+    names=["structure"],
+    values=np.array([[0], [1], [2], [3]]),
+)
```

Comment:
I would not repeat the documentation (`selected_samples can be a variable of `Labels` or `TensorMap` type.`), but explain what is going on and why

```suggestion
# The first way to select samples is with a `Labels` selection. In this case, only samples 
# matching the labels in this selection will be used
selection = Labels(
    names=["structure"],
    values=np.array([[0], [2], [3]]),
)
```